cinema/serializers.py

```diff
@@ -1 +1,79 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("first_name", "last_name", "full_name")
```

Comment:
2. At the same time movie detail endpoint should provide complete information about the genres and actors.
from README. We will not see actors' ids.